back/app/controllers/web_api/v1/projects_controller.rb

```diff
@@ -171,6 +171,19 @@ def votes_by_input_xlsx
     end
   end
 
+  def reset_participation_data
+    project = Project.find(params[:id])
+    authorize project
+
+    ParticipantsService.new.reset_participation_data(project)
```

Comment:
```suggestion
    ParticipantsService.new.reset_participation_data(@project)
```
and add `reset_participation_data` to the list of actions here:
https://github.com/CitizenLabDotCo/citizenlab/blob/0a63ebf93ae73a0f6734fd20b9060769ff673165/back/app/controllers/web_api/v1/projects_controller.rb#L4